src/core/colorspace.js

```diff
@@ -806,6 +862,32 @@ class DeviceRgbaCS extends ColorSpace {
   isPassthrough(bits) {
     return bits === 8;
   }
+
+  fillRgb(
+    dest,
+    originalWidth,
+    originalHeight,
+    width,
+    height,
+    actualHeight,
+    bpc,
+    comps,
+    alpha01
+  ) {
```

Comment:
For consistency, can we please copy the assert used at the top of the [base-class `fillRgb` method](https://github.com/mozilla/pdf.js/blob/e1f9fa4ea54246bbfa974ac75720793b9ff9fd12/src/core/colorspace.js#L130-L151) here as well?
```suggestion
  ) {
    if (typeof PDFJSDev === "undefined" || PDFJSDev.test("TESTING")) {
      assert(
        dest instanceof Uint8ClampedArray,
        'DeviceRgbaCS.fillRgb: Unsupported "dest" type.'
      );
    }
```